src/front/app/components/Create/Create.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component, PropTypes } from 'react';
+import { reduxForm } from 'redux-form';
+
+const validate  = (values) => {
+    const errors = {};
+
+    if (!values.title) {
+        errors.title = 'Title is required';
+    }
+
+    if(!values.description) {
+        errors.description = 'Description is required';
+    }
+
+    return errors;
+};
+
+const EnabledCheckbox = ({ text }) => (
```

Comment:
Enabled and Disabled in this case is Checkbox state, not the kind of checkbox,
as from the **scss** it is just the modificator *_disabled* (enabled by default)
as from the **react component** it is property (*isDisabled* for example)